app/domain/errors.rb

```diff
@@ -474,6 +474,16 @@ module AuthnJwt
         code: "CONJ00098E"
       )
 
+      MissingIdentity = ::Util::TrackableErrorClass.new(
+        msg: "Identity is empty or not found.",
```

Comment:
Can we give the identity an identifier? I;m giessing this is the JWT identity??
 ```suggestion
        msg: "JWT identity is empty or was not found.",
```